src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -480,18 +480,7 @@ private void walk(TypedVar symbol, boolean isDefault) {
     }
 
     private void walkDefaultInterface(FunctionType ftype) {
-      for (String propName : ftype.getOwnPropertyNames()) {
-        if (propName.equals("prototype")) continue;
-        JSType pType = ftype.getPropertyType(propName);
-        // Here we assume the enum is exported and handled separately.
-        if (pType.isEnumType()) continue;
-        emit("var");
-        emit(propName);
-        emit(":");
-        visitType(pType);
-        emit(";");
-        emitBreak();
-      }
+      visitNamespaceLikeType(ftype);
     }
```

Comment:
This is a clean up for a previous PR (`visitNamespaceLikeType` was added by not used), not directly related with the change.